src/leader_scheduler.rs

```diff
@@ -4,6 +4,7 @@
 use crate::bank::Bank;
 
 use crate::entry::{create_ticks, Entry};
+use crate::replay_stage;
```

Comment:
These module dependencies seem backwards. I'd expect stages to reference the leader_scheduler module and never the other way around.

Reply:
moved BLOCK_TICK_COUNT to leader_scheduler.rs, so that replay stage is dependent on leader_scheduler, not the other way around.